server/load_store.go

```diff
@@ -97,9 +200,10 @@ func LoadStoreManager(ctx context.Context, cfg CLIConfig, log logging.Logger, m
 		log.Info("Using memstore backend for EigenDA")
 		eigenDA, err = memstore.New(ctx, verifier, log, cfg.EigenDAConfig.MemstoreConfig)
 	} else {
+		// EigenDAV1 backend dependency injection
 		var client *clients.EigenDAClient
-		log.Info("Using EigenDA backend")
-		client, err = clients.NewEigenDAClient(log.With("subsystem", "eigenda-client"), daCfg.EdaClientConfig)
+		log.Warn("Using EigenDA backend.. This backend type will be deprecated soon. Please migrate to V2.")
```

Comment:
> Using EigenDA backend..

Is the double `.` intentional?

Reply:
yes

Reply:
What does `..` imply?

Reply:
just a segway to the next sentence - individual bias